frontend/dockerfile/dockerfile_outline_test.go

```diff
@@ -58,7 +59,35 @@ FROM third AS target
 COPY --from=first /etc/passwd /
 
 FROM second
-`)
+`,
+		`
+ARG inherited=server
+ARG inherited2=server2
+ARG unused=abc${inherited2}
+# sfx is a suffix
```

Comment:
curious why not just name the arg suffix

Reply:
just maintained the tradition from L40 :)